.gitignore

```diff
@@ -54,3 +54,9 @@ _UpgradeReport_Files/
 Backup*/
 UpgradeLog*.XML
 UpgradeLog*.htm
+
+# CMake files
+CMakeCache.txt
+CMakeFiles
+Makefile
+cmake_install.cmake
```

Comment:
Cmake should rather create its files in a directory like `build/`.

Reply:
How would you do that? I can specify where the build is output of the build is put to (already to build), but intermediary files (like `Makefile` and `CMakeCache.txt`) are written to `.`

Otherwise the build also needs to change directories between steps:

```bash
cmake
cd build
make
```